ui/js/filters.js

```diff
@@ -121,3 +121,15 @@ treeherder.filter('escapeHTML', function() {
         return '';
     };
 });
+
+treeherder.filter('getRevisionUrl', function() {
+    return function(revision, projectName) {
+        if (revision) {
+            var resultsetUrl = 'https://treeherder.mozilla.org' + '#/jobs?repo=' + projectName;
+            resultsetUrl = resultsetUrl + revision;
```

Comment:
This isn't right, it yields results like: https://treeherder.mozilla.org/#/jobs?repo=tryb015adea7c85

1. Revision also needs to be specified explicitly. 
2. Don't hardcode https://treeherder.mozilla.org, instead pull in thServiceDomain from providers. Lots of examples of this type of pattern elsewhere in the code. :)